src/test/regress/output/multi_copy.source

```diff
@@ -304,6 +304,28 @@ SELECT count(*) FROM pg_dist_shard WHERE logicalrelid = 'lineitem_copy_append'::
      5
 (1 row)
 
+-- Test round robin shard policy
```

Comment:
I think it will always use the first port number for the host, so you would see a notably different result from round-robin, but perhaps that's not really something we can rely on (e.g. if we change pg_worker_list.conf to a table, things might change), so maybe better to leave it out.